JT/01_environment_setup/Env_setup_for_Mac.adoc

```diff
@@ -0,0 +1,38 @@
+= Software Installation for Mac
+:toc:
+:icons: font
+
+=== SDK
+
+. https://www.oracle.com/java/technologies/downloads/#jdk21-mac[*Download*] *and Java SE Development Kit:*
```

Comment:
Let's not use oracle
https://adoptium.net/ is a good alternative

Reply:
Done